protocol/state.re

```diff
@@ -16,6 +16,9 @@ type t = {
   // shared consensus
   block_height: int64,
   last_block_hash: BLAKE2B.t,
+  // List of state_root_hash'es computed by this node from a last known good state
+  // This will later be validated against the main chain
```

Comment:
Nonblocking: not sure the wording here is right, 

> validated against the main chain

Reply:
Is `validated` the problematic word? I'm not sure what's better here